### ISSUE TITLE
WindTunnel working with Quantum

### ISSUE THREAD

**Issue by sumitv-007** [Project-Quantum/Quantum-Starter-Kit#24]
We for sure need one example of Quantum script working with Wind Tunnel. This is particularly important as majority of APAC customers have moved or are moving to Quantum.

**Comment by genesisthomas** [comment]
agreed. here's a java + wind tunnel sample to get started: https://developers.perfectomobile.com/display/PD/Add+performance+testing+to+your+script
